esp32_s3_eye/include/bsp/esp32_s3_eye.h

```diff
@@ -311,6 +318,82 @@ esp_err_t bsp_leds_init(void);
  */
 esp_err_t bsp_led_set(const bsp_led_t led_io, const bool on);
 
+/**************************************************************************************************
+ *
+ * I2S audio interface
+ *
+ * There is only one device connected to the I2S peripheral
+ *  - MEMSensing Microsystems MSM261S4030H0: 48kHz, 24bit mono digital microphone
+ **************************************************************************************************/
+
+/**
+ * @brief Sample rate of MSM261S4030H0
+ */
+#define BSP_MIC_SAMPLE_RATE (48000u)
+
+/**
+ * @brief ESP32-S3-EYE I2S pinout
+ *
+ * Can be used for i2s_std_gpio_config_t and/or i2s_std_config_t initialization
+ */
+#define BSP_I2S_GPIO_CFG       \
+    {                          \
+        .mclk = GPIO_NUM_NC,   \
+        .bclk = BSP_I2S_SCLK,  \
+        .ws = BSP_I2S_LCLK,    \
+        .dout = GPIO_NUM_NC,   \
+        .din = BSP_I2S_DIN,   \
+        .invert_flags = {      \
+            .mclk_inv = false, \
+            .bclk_inv = false, \
+            .ws_inv = false,   \
+        },                     \
+    }
+
+/**
+ * @brief Mono Simplex I2S configuration structure
+ *
+ * This configuration is used by default in bsp_audio_init()
+ */
+#define BSP_I2S_SIMPLEX_MONO_CFG                        \
```

Comment:
We are using format with brackets in IDF for deafult config. Example: `BSP_I2S_SIMPLEX_MONO_CFG() { /`